server/metadata/cluster.go

```diff
@@ -16,94 +16,77 @@ package metadata
 
 import (
 	"context"
-	"time"
 
 	"github.com/google/uuid"
-	jsoniter "github.com/json-iterator/go"
 	"github.com/tigrisdata/tigris/errors"
 	"github.com/tigrisdata/tigris/keys"
 	"github.com/tigrisdata/tigris/server/transaction"
-	ulog "github.com/tigrisdata/tigris/util/log"
 )
 
+// ClusterMetadata keeps cluster wide metadata.
+type ClusterMetadata struct {
+	ID uuid.UUID
+}
+
 // ClusterSubspace is used to store metadata about Tigris clusters.
 type ClusterSubspace struct {
 	metadataSubspace
 }
 
 var (
-	clusterVersion     = []byte{0x01}
 	clusterID          = uuid.Nil.String()
 	clusterMetadataKey = "key"
```

Comment:
"key" may not that intuitive, should we follow the same convention as we have for remaining and call it "cluster" or something similar?

Reply:
We need it earlier when it was a single namespace, but probably not now. But I'd suggest having it just in case? as it allows expanding the key.

Reply:
yeah, keeping it.